Week-3/Assignment-2/index.js

```diff
@@ -0,0 +1,35 @@
+const obj = {
+    items: [1, 2, 3],
+    getItems(newItem1, newItem2, newItem3) {
+        console.log([...this.items, newItem1, newItem2, newItem3]);
+    },
+    getItemsAll(...newItems) {
+        console.log([...this.items, ...newItems]);
+    }
+}
+
+const getItems = obj.getItems;
+const getItemsAll = obj.getItemsAll;
+
+// bind - accepts positional arguments and creates a copy of function for future reference & permanently binds the passed object with it
+const getItemsBind = obj.getItems.bind(obj, 0, 0, 0);
```

Comment:
Can we also use a different object to show how these functions work ?

Reply:
Thanks @sairamin, I've now added separate objects to demonstrate call, bind & apply, please review & merge.